dev-python/rtslib-fb/Manifest

```diff
@@ -1 +1,2 @@
 DIST rtslib-fb-2.1.74.tar.gz 73919 BLAKE2B d16527ed1efae379aa976929d8e6dd8a13cb1d47eb8efe8897ee495b0864225f520b42bd27fe502a1fad11369d20690b9751e54ae7a328c60917144114358d24 SHA512 b0b0722a5a7e10af698ae0570a65b51d7f091bfcc370850dacb3087471ca98bd229b4a5ed73760b7ed5bbee1b53dae4781cc2b3d5e7a1ff8ce37c5c144477852
+DIST rtslib-fb-2.1.75.tar.gz 74188 BLAKE2B 13fbeffb0396fa39d17362c05973572feab86eaa6748d4043f7721f3b631e670266dfbaebc26afb70e83547c4a24f149399d7d77725ecb9e9f6ac4ef709306d4 SHA512 b20109fd40cd0d23621bb01f884e302432a1dde98d0147dbd7a2eaf7a724801c77759effb26717b0451c5fb2f75e67287d9c0364c3c7ee3072d08cce259ce616
```

Comment:
.76 seems to be the latest upstream version, could we od that instead?

Reply:
I mean I can bump it to .76 but our SAN's next upgrade cycle is more than a month out so it would be a blind bump.. in the mean time there're probably busted systems out there and we have confirmation that .75 fixes the problem on hand.

Reply:
Sure, it's just unconventional to bump to the non-latest version. I'm not saying no, but some explanation as to why when submitting something non-latest is customary.

Reply:
TBF it was the latest when I submitted the original PR. Though I did forget about it since, so yeah, what would you like for me to do?

Reply:
We're discussing this in the context of a new PR. You didn't need to close the old one - just force-push to it, but done with now.

Anyway, in the here-and-now: just mention the rationale in the version bump commit, thanks.